attachments/models.py

```diff
@@ -148,12 +148,11 @@ def __str__(self):
     def choice_list(self):
         return [ch.strip() for ch in self.choices.split('\n') if ch.strip()]
 
-    @property
-    def model_queryset(self):
+    def model_queryset(self, **kwargs):
         ModelClass = import_class(self.model)
         # Lookup models can provide an @classmethod 'field_model_queryset' to have control over what queryset is used
         if hasattr(ModelClass, 'field_model_queryset'):
-            qs = getattr(ModelClass, 'field_model_queryset')()
+            qs = getattr(ModelClass, 'field_model_queryset')(**kwargs)
```

Comment:
I think we want this wrapped in a try catch

most sites field_model_queryset functions probably dont accept kwargs so we should try with kwargs and without if it errors.